Builds failing with CMake error when installing mysql

Since some days ago, I've been observing that some Launchpad builds for amd64 & arm64 are failing with the following error when building the mysql part:
```
-- Installing: /build/nextcloud/parts/mysql/install/support-files/mysql-log-rotate
CMake Error at support-files/cmake_install.cmake:62 (file):
  file INSTALL given no DESTINATION
Call Stack (most recent call first):
  cmake_install.cmake:472 (include)


make: *** [Makefile:120: install] Error 1
Failed to build 'mysql'.
```

Some examples [here](https://launchpad.net/~nextcloud-snap-dev/+snap/nextcloud-daily-master/+build/3215027/+files/buildlog_snap_ubuntu_focal_amd64_nextcloud-daily-master_BUILDING.txt.gz) and [here](https://launchpad.net/~nextcloud-snap-dev/+snap/nextcloud-daily-master/+build/3215028/+files/buildlog_snap_ubuntu_focal_arm64_nextcloud-daily-master_BUILDING.txt.gz).
Builds for ppc64el arch succeed; see [here](https://launchpadlibrarian.net/868687968/buildlog_snap_ubuntu_focal_ppc64el_nextcloud-daily-master_BUILDING.txt.gz).

The error is also happening now on GitHub runners; see [here](https://github.com/nextcloud-snap/nextcloud-snap/actions/runs/29239175152/job/86780836940).

This means that, right now, we cannot do any release.

I suspect this is related to a package update, as we didn't touch that part lately, though that’s just a hunch.

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -369,7 +369,6 @@ parts:
       - libfontconfig1
 
   mysql:
-    build-snaps: [cmake]
     plugin: cmake
 
     # Get from https://dev.mysql.com/downloads/mysql/
@@ -379,6 +378,7 @@ parts:
       - -DCMAKE_INSTALL_PREFIX=/
       - -DBUILD_CONFIG=mysql_release
       - -DCMAKE_BUILD_TYPE=Release
+      - -DWITH_ROUTER=OFF
       - -DWITH_UNIT_TESTS=OFF
       - -DWITH_EMBEDDED_SERVER=OFF
       - -DWITH_ARCHIVE_STORAGE_ENGINE=OFF
@@ -397,6 +397,7 @@ parts:
       - CC: gcc-10
       - CXX: g++-10
     override-build: |
+      cmake --version
       snapcraftctl build
       # MySQL v8 is massive. Strip it.
       find "$SNAPCRAFT_PART_INSTALL/bin" -type f -exec sh -c 'grep -IL . "$1" || strip --strip-all "$1"' sh "{}" \;

